sass/components/_post.scss

```diff
@@ -1651,6 +1651,98 @@
         opacity: 0.6;
     }
 
+    // styles for utility buttons i.e download, and copy source link in image preview
+    .image-preview-utility-buttons-container {
+        // greater z-index than img
+        position: absolute;
+        z-index: 3;
```

Comment:
is it in the same stacking context? If yes a value of `1` should be sufficient.

Reply:
also this change puts the buttons behind the image, will test it with 2, then 3.

Reply:
this is weird, it's happening for some images in test server, but for the same image in my local setup, the buttons stack correctly.

Reply:
cc: @matthewbirtch I’ll fix that.

Reply:
https://github.com/mattermost/mattermost-webapp/blob/master/sass/components/_files.scss#L222 will make `z-index: 3`

Reply:
That seems to have fixed the issue.